routes/adminRoutes.js

```diff
@@ -62,7 +62,7 @@ router.put('/team/:teamId', async (req, res) => {
   try {
     const id = req.params.teamId;
     const { name, orgId, note } = req.body;
-    Exceptions.throwIfNull({ id, name });
+    Exceptions.throwIfNull({ id, name, note });
```

Comment:
Why is `note` for teams required?

Reply:
I don't really see a need for it.  If we have a consensus, we can remove it.